schemas/context/profile-work-details.schema.json

```diff
@@ -17,27 +17,75 @@
     "profile-work-details": {
       "properties": {
         "xdm:workAddress": {
-          "title": "Work address",
+          "title": "Work Address",
           "$ref": "https://ns.adobe.com/xdm/common/address",
           "description": "A work postal address."
         },
+        "xdm:otherAddress": {
+          "title": "Other Address",
+          "$ref": "https://ns.adobe.com/xdm/common/address",
+          "description": "Other postal address."
+        },
         "xdm:workEmail": {
-          "title": "Work email",
+          "title": "Work Email",
           "$ref": "https://ns.adobe.com/xdm/context/emailaddress",
           "description": "A work email address."
         },
         "xdm:workPhone": {
-          "title": "Work phone",
+          "title": "Work Phone",
           "$ref": "https://ns.adobe.com/xdm/context/phonenumber",
           "description": "Work phone number."
         },
+        "xdm:otherPhone": {
```

Comment:
Need to have additional "data type" which captures information for a scope beyond a particular type.

Reply:
Even if we create a new data type, it would still have the attributes of a phonenumber which are already existing in the "https://ns.adobe.com/xdm/context/phonenumber" data type.  Would creating another object as I did and just referring the same datatype cause any issues ? I just want to understand this a little bit more. Please clarify.